src/Components/ArticlesList.jsx

```diff
@@ -0,0 +1,37 @@
+import { useEffect, useState } from "react";
+import { getAllArticles } from "../utils/api.js";
+import ArticlesCSS from "../Components/styles/ArticleList.module.css";
+
+export default function ArticlesList() {
+  const [articles, setArticles] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    getAllArticles().then((articles) => {
+      setArticles(articles);
+      setIsLoading(false);
+    });
+  });
+
+  if (isLoading) {
+    return (
+      <div>
+        <h3>Loading...</h3>
+      </div>
+    );
+  }
+
+  return (
+    <section>
+      <h2 className={ArticlesCSS.ArticleH2}>All Articles</h2>
+      <ul className={ArticlesCSS.ArticleUL}>
+        {articles.map((article) => (
+          <section key={article.article_id}>
+            <p>{article.title}</p>
+            <img src={article.article_img_url} alt={article.title}></img>
+          </section>
```

Comment:
I'd consider showing a little more information here, looking at similar news article websites for inspiration, they usually contain the date etc.. something to think about when you flesh this project out 💡